engines/scumm/script_v2.cpp

```diff
@@ -1240,7 +1240,7 @@ void ScummEngine_v2::o2_startScript() {
 	// a permanent resident.
 	// Our fix is simply to prevent the Cutscene playing, if the lab has already been stormed
 	if (_game.id == GID_MANIAC) {
-		if script == MM_SCRIPT(150) {
+		if (script == MM_SCRIPT(150)) {
 			if VAR(MM_VALUE(104, 120) == 1)
```

Comment:
@einstein95 you still have one typo left here

Reply:
Right, I really shouldn't edit using Github's web interface...